src/main/java/com/nhnacademy/hexashoppingmallservice/exception/CartException/CartAlreadyExistException.java

```diff
@@ -0,0 +1,7 @@
+package com.nhnacademy.hexashoppingmallservice.exception.CartException;
+
+public class CartAlreadyExistException extends RuntimeException {
+    public CartAlreadyExistException(String message) {
```

Comment:
예외 처리를 하는 advice가 있는지?
```suggestion
    public CartAlreadyExistException(String message) {
```